acktools/log.py

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/python3
 
-# Copyright (c) 2005-2022 Citrix Systems Inc.
```

Comment:
These need not to remove as my understanding.

Reply:
I have recovered the copyright for Citrix